ops/README.md

```diff
@@ -2,23 +2,29 @@
 
 ## The application
 
-Let's say you have a multi-tier blogging application.
+You are working as a sysadmin in a new company. This company is about to release a new blog, via a blogging application developped internally.
```

Comment:
This company is about to release a retail webapp.

> autant parler d'une app à nous

Reply:
Si t'es pas dans le métier, retail webapp ça parle pas. Au moins blog, tout le monde voit ce que c'est.